backend/src/main/java/com/eyetrade/backend/model/entity/User.java

```diff
@@ -1,30 +1,29 @@
 package com.eyetrade.backend.model.entity;
 
 import com.eyetrade.backend.constants.Role;
-import lombok.Data;
-import org.hibernate.annotations.GenericGenerator;
+import lombok.Getter;
+import lombok.Setter;
 
 import javax.persistence.*;
 import javax.validation.constraints.Email;
 import javax.validation.constraints.NotEmpty;
 import javax.validation.constraints.NotNull;
 
-import static com.eyetrade.backend.constants.GeneralConstants.ID_LENGTH;
-
 /**
  * Created by Emir Gökdemir
  * on 12 Eki 2019
  */
-@Data
+
+@Getter
+@Setter
 @Entity
 @Table(name="users", uniqueConstraints = {@UniqueConstraint(columnNames = {"email"})})
 public class User {
 
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO, generator = "system-uuid")
-    @GenericGenerator(name = "system-uuid", strategy = "uuid2")
-    @Column(name = "id", length = ID_LENGTH)
-    private String identifier;
```

Comment:
Only auto generated as enumerated way, also because uuid2 is a standart of generation id number, It is more acceptable .